changelog.md

```diff
@@ -2,6 +2,38 @@
 
 
 
+## Changes affecting backward compatibility
```

Comment:
that section was requested by @dom96 in another PR IIRC

Reply:
EDIT: actually it was this very PR see https://github.com/nim-lang/Nim/pull/16351#discussion_r545386949